packages/components/src/components/Dialog/DialogHeader.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { EDSStyleSheet } from "../../styling";
+import { TextChildren, Typography } from "../Typography";
+import { View } from "react-native";
+import { useStyles } from "../../hooks/useStyles";
+
+export const DialogHeader = (props: TextChildren) => {
+    const styles = useStyles(themeStyles);
+    return <View style={styles.header}>
+        <Typography variant="h6" style={styles.title} >{props.children}</Typography>
+    </View>
+
+}
+
+const themeStyles = EDSStyleSheet.create(theme => ({
+    header: {
+        height: 49,
```

Comment:
Maybe this should be dynamic instead.
Seeing as this is a standalone component in contrast to a prop on the `Dialog` component, it should provide maximum flexibility and therefore allow for stuff like multiple lines of text or just any container.

Reply:
I haven't seen any designs where the header is more than 50px tall (49 + border). I don't think giving the flexibility to adjust it is a good idea.

We could make title a prop instead but let's save that rewrite for later

Reply:
Added it as a value in mastertoken though

Reply:
> I haven't seen any designs where the header is more than 50px tall (49 + border). I don't think giving the flexibility to adjust it is a good idea.

To be more clear, if a developer creates a title long enough to cover 2 lines, it should send a clear message that their title suck, and they should reduce the length of it.

Reply:
Like this 
![image](https://github.com/equinor/mad/assets/22524448/b3ae7d81-838d-484e-a39a-b23594232f26)

Reply:
In that case, we should just drop the `Dialog.Header` component and just have `title` be a prop on the Dialog component instead. You mentioned this, but I actually think this change should be done for this PR and not the future. Why bother introducing a component that we know will be removed? It could create frustration with someone potentially implementing this component while we fix it